airtime_mvc/public/js/airtime/login/login.js

```diff
@@ -4,6 +4,6 @@ $(window).load(function() {
 
 $(document).ready(function() {
     $("#submit").click(function() {
-        $.cookie("airtime_locale", $("#locale").val(), {path: '/'});
+        Cookies.set('airtime_locale', $('#locale').val(), {path: '/'});
```

Comment:
Single quotes are inconsistent with the rest of the JavaScript

Reply:
Noted